core/src/main/java/org/apache/gravitino/storage/relational/service/MetadataObjectService.java

```diff
@@ -57,6 +59,18 @@ public class MetadataObjectService {
 
   private static final Logger LOG = LoggerFactory.getLogger(MetadataObjectService.class);
 
+  public static final Map<MetadataObject.Type, Function<List<Long>, Map<Long, String>>>
```

Comment:
Why do you make it public?

Reply:
`RoleMetaService` also uses it. I can lower the access level and use the package level.

Reply:
Changed